porcupine/plugins/filetypes.py

```diff
@@ -236,7 +236,7 @@ def _sync_filetypes_menu(event: object = None) -> None:
     filetype_name: str = ""
```

Comment:
This variable can now be None:
```suggestion
    filetype_name: str | None = ""
```

Unfortunately it doesn't seem to be possible to return the correct type from `.get()`, so that mypy would point this out.